clang/lib/Driver/OffloadBundler.cpp

```diff
@@ -1181,13 +1206,17 @@ class ArchiveFileHandler final : public FileHandler {
             OS << ChildFileName << "\n";
           } else if (Mode == OutputType::Object) {
             // Extract the bundle to the output file in single file mode.
-            if (Error Err = OFH.ReadBundle(OS, *Buf))
+            if (Error Err = FHP->ReadBundle(OS, *Buf))
               return Err;
           } else if (Mode == OutputType::Archive) {
+            auto ChildNameOrErr = C.getName();
```

Comment:
Why do we need this?
ChildNameOrErr?

Reply:
This corresponds to line 1142 from the original code.